client/src/components/application/Stats/Stats.vue

```diff
@@ -183,6 +194,9 @@ export default {
       let currentPage = '&currentpage=' + this.currentPage
       let pageSize = '&perpage=' + this.perPage
       let url = '/api/v1/requests/stats' + params + currentPage + pageSize
+      if (this.myStats) {
+        url += '&myStats=True'
```

Comment:
Where does this URL parameter get consumed?

Reply:
The url is consumed in namex. If myStats=True, userId is a condition of the stats query.

Reply:
OK, thanks.

I see `numRecords` is read from the API response on line 219, which answers the overall question 👍